weatherman.py

```diff
@@ -67,16 +61,14 @@ def main():
                 f"Invalid date '{parameters[iteration][1]}'. "
                 f"Please enter a valid command: "
             ).split(" ")
-            parameters, args = getopt.getopt(parameters, allowed_parameters)
+            parameters, args = getopt.getopt(parameters, ":e:a:")
             path = constants.WEATHER_FILES_DIR
             if args:
-                path, parameters = (
-                    args[0],
-                    getopt.getopt(args[1:], allowed_parameters)[0],
-                )
+                path, parameters = args[0], getopt.getopt(args[1:], ":e:a:")[0]
             iteration = 0
             continue
         iteration += 1
+        print()
```

Comment:
Is it doing anything specific?

Reply:
it just add a new line to make it easier to differentiate between each report